content/posts/2022-02-02-decreased-backend-tests-step/index.md

```diff
@@ -0,0 +1,172 @@
+---
+title: How we decreased backend tests step execution time?
+author: Gustavo Rodrigues
+twitter: gustahrodrigues
+layout: post
+lang: en
+path: /blog/how-we-decreased-pipeline-time
+date: 2022-02-02
+comments: true
+---
+
+At Loadsmart we use [CircleCI](https://circleci.com/) as a pipeline tool to assert the new code’s correctness and ensure high-quality code before integration.
+
+Some time ago we realized that the pipeline for one of our largest codebases was taking about 45min (about 40 or 50), of which 22min was spent to execute the backend test step.
+It means that, for each PR, developers should wait for at least 45min to get feedback about their changes, delaying the time to deliver code to production.
+
+We had created a Working Group initiative to improve it. This post aims to share our journey to improve our backend test step execution time.
+
+## What is a Working Group?
+
+Working Groups are a great format when it comes to solve multi-squad tech debts, as they are short-lived, temporary, diverse, and they are formed to solve a cross-organization issue. Cool, right? Would you like to know more about Loadsmart Engineering culture? [Check it out](https://engineering.loadsmart.com/blog/our-engineering-culture).
```

Comment:
I think it would be interesting to cite how many engineers participated in this project, and from what backgrounds they came.